### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Veteran Facing Form - Form 20-10206 FOIA / PA Request

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#66596]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** Look at the heading structure on the entry page to the form, and see if there's a way to make the "Start your request" link a little more findable. Screen reader users often navigate the page by heading, looking for the heading that seems most likely to describe the action they want and jumping to that. Right now, the closest heading to the start link is "Who can submit this request?" which doesn't seem like a likely match.
   
   @humancompanion-usds's question at the meeting today was 100% the right one---how have other forms solved this? And looking at other forms... it looks like they mostly haven't! Looking at other recent products we've reviewed, I'm not seeing any examples of a good heading structure post-auth that helps with the findability of the start link. (Example: [21-0972](https://staging.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/introduction) has the same issue.) Not sure how no one has noticed this before!
   
   The impact here isn't huge (this is a "should" and not a "must"), mainly because screen reader users have multiple ways of navigating the page and finding what they're looking for. But this is definitely an opportunity to establish a pattern that we can replicate elsewhere.
- **Consider:** For users that log in but aren't LOA3 users, consider giving them an exit ramp to the PDF form when prompting them to go through the identity verification process. The main accessibility concern here is that screen reader users frequently struggle with the identity verification processes for ID.me and Login.gov, and may prefer to delay going through those processes if another way to complete the form exists. (Screen reader users are also likely to hate filling out the PDF, but it's hard to say which is the lesser of two evils for any given user!)
- **Must:** Even when content is technically accessible, it may be functionally hidden from screen reader users based on anticipated behavior when interacting with a form. Typically, screen reader users aren't going to have their computers announce the whole page for every page of a form. Instead, they'll probably tab directly into each input --- which means any helper text that's not programmatically associated with the inputs won't get announced to them. Especially when it's information that's critical for understanding a question or likely to impact how a user might answer it (eg. the warning alert about processing fees), you want to be extra careful about how the page is structured to ensure that it's announced.
   
   @jeana-adhoc has recently put together a document outlining all of the use cases around this kind of text and the best ways to ensure it's announced. Definitely sync with her on this!
- **Consider:** On step 4, the radio buttons about processing fees conditionally reveal new inputs. I think given the context here (keeping the new inputs close to both the radio buttons and the accordions with more information) this is a reasonable use case for conditionally revealing something new. But those kinds of conditional reveals aren't super great for assistive technology users, so I'd encourage you to explore another version of this as a one-thing-per-page interaction and just see if there's a way that it makes sense. If it doesn't make sense (inputs too far removed from context, or it's just annoying to the user) then keep the conditional-reveal while following the [guidance around mitigating the AT interaction issues](https://design.va.gov/patterns/wizards#retrofit-this-pattern).

**Comment by rachelshearerux** [comment]
Late to closing this ticket, but these have been resolved, except for displaying a PDF link on the intro page. 
 
> Should: Look at the heading structure on the entry page to the form, and see if there's a way to make the "Start your request" link a little more findable..
- We've added an H2 “Start your form” just above the "Sign in" CTA

> Must: Even when content is technically accessible, it may be functionally hidden from screen reader users based on anticipated behavior when interacting with a form. Typically, screen reader users aren't going to have their computers announce the whole page for every page of a form. Instead, they'll probably tab directly into each input
- The particular page mentioned here (FOIA Fees) is no longer part of this flow, but the feedback is very much heard and has been and will continue to be our goal. 

> Consider: For users that log in but aren't LOA3 users, consider giving them an exit ramp to the PDF form when prompting them to go through the identity verification process... 
- We did explore a path where users who fail authenticating are redirected to a page that explains identity verification AND offers a link to download the PDF. Ultimately we will not redirect users to another page where the PDF download link would've lived, rather we've adjusted our flow to find better alignment with what exists elsewhere on VA.gov. When someone signs in to a non-LOA3 account, they'll be redirected to the intro page and anchored to the H2 "Start your form", just below that the standard "sign in" component will explain identity verification with a CTA to go do that. Once they've done that, they redirect again to the same place, where they see the CTA "Start your request". 
- There’s a larger need for a standardized flow when these authentication errors come up for users with lower-level authentication. This [experimental design ticket ](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2284) sparked a lot of conversation and CAIA is going to continue to explore this, I’ll happily pass along this feedback to them!

>Consider: On step 4, the radio buttons...
- FOIA Fees are no longer part of this flow
